### DISCUSSION TITLE
3d viewer does not work

### DISCUSSION THREAD

**Discussion by gerroon** [JacquesLucke/animation_nodes#1928] [Q&A] [answered]
Hi

I am guessing that this is broken in 4.0 I am using 2.3.0 on Windows When I plug 3d viewer to any kind of matrix I do nto see anything drawn in the viewport.

thanks

**Comment by OmarEmaraDev** [comment, marked as answer]
We already fixed that, so the daily builds should work. The problem is that we are not able to get daily builds to work on Github for Windows, so they are not up to date. I will try to fix that as soon as possible. Meanwhile, if you are able to able to build from source, try that.

**Reply by gerroon** [reply]
Thanks I will wait for the windows builds since I cant build it myself.

**Reply by OmarEmaraDev** [reply]
@gerroon New builds are now up.
